src/ConstructiveSolidGeometry/SurfacePrimitives/ConeMantle.jl

```diff
@@ -34,6 +34,12 @@ radius_at_z(hZ::T, rBot::T, rTop::T, z::T) where {T} = iszero(hZ) ? rBot : rBot
 radius_at_z(cm::ConeMantle{T,T}, z::T) where {T} = cm.r
 radius_at_z(cm::ConeMantle{T,Tuple{T,T}}, z::T) where {T} = radius_at_z(cm.hZ, cm.r[1], cm.r[2], z)
 
+get_r_limits(cm::ConeMantle{T,T}) where {T} = cm.r, cm.r
+get_r_limits(cm::ConeMantle{T,Tuple{T,T}}) where {T} = cm.r[1], cm.r[2]
```

Comment:
Does not seem to be needed.
Please delete or comment this out if it is not needed